src/main/java/net/sf/jabref/cli/ArgumentProcessor.java

```diff
@@ -208,7 +211,7 @@ private boolean exportMatches(List<ParserResult> loaded) {
                             matches);
                 } catch (Exception ex) {
                     System.err.println(Localization.lang("Could not export file") + " '" + data[1] + "': "
-                            + ex.getMessage());
+                            + ExceptionUtils.getStackTrace(ex));
```

Comment:
I looked at all other cases and there is either a logger attached or a specific Exception, e.g. a SaveException/JabRef Exception catched and the message printed, which is fine enough.
Only this two line were problematic, as the getMessage returned null